argus/recipes/cloud/windows.py

```diff
@@ -41,9 +41,16 @@ def wait_for_boot_completion(self):
         LOG.info("Waiting for first boot completion...")
         self._backend.remote_client.manager.wait_boot_completion()
 
+    def set_mtu(self):
+        set_mtu_cmd = ('netsh interface ipv4 set subinterface "Ethernet" '
+                       'mtu=1400 store=persistent')
+        LOG.info("Setting the MTU.")
+        stdout = self._backend.remote_client.run_command_verbose(set_mtu_cmd)
```

Comment:
The `run_command_verbose` will catch any exception and also log any output of the command.